api/controllers/environment.go

```diff
@@ -28,7 +32,7 @@ func (e *EnvironmentController) CreateEnvironment(request *restful.Request, resp
 		// todo: handle error
 	}
 
-	environmentModel, err := environment.Read()
+	environmentModel, err := environment.Model()
```

Comment:
ahhh I see why you did `Model` when put in context with the other `Get` methods....